e2e/test/scenarios/sharing/public-sharing-embed-button-behavior.cy.spec.js

```diff
@@ -284,11 +441,24 @@ function createPublicResourceLink(resource, id) {
 }
 
 function visitResource(resource, id) {
-  if (resource === "card") {
+  if (resource === "card" || resource === "question") {
     visitQuestion(id);
   }
 
   if (resource === "dashboard") {
     visitDashboard(id);
   }
 }
+
+function enableEmbeddingForResource({ resource, id, isDirty = false }) {
+  const endpoint = resource === "question" ? "card" : "dashboard";
+  cy.request("PUT", `/api/${endpoint}/${id}`, {
+    enable_embedding: true,
+    embedding_params: isDirty
+      ? {
+          test: "locked",
```

Comment:
We don't need a valid parameter here, since we check whether to display `Publish`, `Discard`, and `Unpublish` based on the difference between the `embedding_params` that the Question/Dashboard already has, and the new `embedding_params` that we're generating in the static embed modal